.github/workflows/benchmark.yml

```
@@ -159,7 +166,7 @@ jobs:
         with:
           service_account_key: ${{ secrets.GCP_SA_KEY }}
           project_id: ${{ secrets.GCP_PROJECT_ID }}
-          zone: ${{ vars.GCP_ZONE }}
+          zones: ${{ vars.GCP_ZONES }}
```

actionlint (1.7.12)

[error] input "zones" is not defined in action "Cleanup running runner" defined at "./.github/actions/cleanup-runner". available inputs are "instance_label", "project_id", "service_account_key"

(action)